discordpp/categories/guild.hh

```diff
@@ -512,20 +486,17 @@ sptr<const std::string> render_target() override {
 #include <discordpp/macros/defineCallOpen.hh>
 protected:
 sptr<const std::string> render_target() override {
-    std::string out = fmt::format("/guilds/{}/members/search", *_guild_id);
-    bool first = true;
     if (!_guild_id) {
         throw std::logic_error("Search Guild Members needs a Guild ID");
     }
+    std::string out = fmt::format("/guilds/{}/members/search", *_guild_id);
     if(!_query) {
         throw std::logic_error("Search Guild Members needs a Query");
     } else {
-        out += fmt::format("{}query={}", first ? "?" : "&", *_query);
-        first = false;
+        out += fmt::format("?query={}", *_query);
```

Comment:
Add this to `out`'s declaration

Reply:
so like this "std::string out = fmt::format("/guilds/{}/members/search?query={}", *_guild_id, *_query);"?

Reply:
Yes

You can use backticks for inline code fyi: `